proto/process/agent.proto

```diff
@@ -841,3 +847,29 @@ message DNSStats {
 	uint64 dnsFailureLatencySum = 3;
 	map<uint32, uint32> dnsCountByRcode = 4;
 }
+
+message HTTPKey {
+	Addr source = 1;
+	Addr dest = 2;
+}
+
+message HTTPStatsByPath {
+	map<string, HTTPStats> stats = 1;
+}
+
+message HTTPStats {
+	repeated Data statsByResponseStatus = 1;
+
+	message Data {
+		uint32 count = 1;
+		DDSketch latencies = 2;
+	}
+}
+
+enum HTTPResponseStatus {
```

Comment:
We are not going to track individual codes? There aren't that many. `statsByResponseStatus` would likely be a map if we did track them individually.

Reply:
Not at first, though we could add that later if we want

Reply:
The difference between some codes, like `400` and `404` would be really useful to track.

Reply:
That's true, but I'd like to get some benchmarks for memory usage before splitting up the data points into smaller chunks